src/main/resources/com/uber/jenkins/phabricator/credentials/ConduitCredentialsImpl/config.jelly

```diff
@@ -4,6 +4,9 @@
     <f:entry title="Phabricator URL" field="url">
         <f:textbox/>
     </f:entry>
+    <f:entry title="Phabricator Gateway (optional)" field="gateway">
```

Comment:
Can you add some help text on what a "Gateway" is (and  why it's optional)? I don't imagine many people are going to intuit this.